Objective-C/CBLDatabase.mm

```diff
@@ -1043,9 +1044,8 @@ - (bool) resolveConflictInDocument: (NSString*)docID
             resolvedDoc = [conflictResolver resolve: conflict];
             
             if (resolvedDoc && resolvedDoc.id != docID) {
-                [NSException raise: NSInternalInconsistencyException
-                            format: @"Resolved docID '%@' is not matching with docID '%@'",
-                 resolvedDoc.id, docID];
+                CBLWarn(Sync, @"Resolved docID '%@' is not matching with docID '%@'",
```

Comment:
Is this messages in synced with the other platforms? Maybe ...

"The document ID of the resolved document is not matching with the document ID of the conflicted document."